tutorial/debezium-cassandra-init/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM cassandra:3.11
+
+RUN apt-get update && \
+    apt-get install -y wget curl sudo vim nano git gnupg
```

Comment:
Are all the utilities but `wget` necessary?

Reply:
Removing the unused ones, we don't need them now.